src/components/ButtonText/style.scss

```diff
@@ -0,0 +1,17 @@
+.buttonText {
```

Comment:
class mendingan `kebab-case`

Reply:
kenapa kebab-case bang?

Reply:
karena nanti kan bakal dicompile html, css, js biasa jadi biar konsistem sama html, css yang normal

https://stackoverflow.com/questions/7560813/why-are-dashes-preferred-for-css-selectors-html-attributes

Reply:
tambahin props onPressed juga ya

Reply:
nanti onPressed itu jadi function bang?

Reply:
mksd saya onPress di props atau dimana nya bang?